tcf_core/settings/base.py

```diff
@@ -7,7 +7,10 @@
 import environ
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+BASE_DIR = os.path.dirname(
```

Comment:
Is this the autoformatting again? might wanna discard these changes hmm

Reply:
I think they came up right after I opened my new branch so I think it is autoformatting I'll discard them and not commit them for future issues/features